src/models/User.js

```diff
@@ -15,6 +79,14 @@ export default (sequelize, DataTypes) => {
       foreignkey: 'userId',
       onDelete: 'CASCADE'
     });
+    User.hasMany(models.user, {
+      foreignKey: 'lineManagerId',
+      onDelete: 'CASCADE'
+    });
+    User.belongsTo(models.user, {
```

Comment:
I think this association is unnecessary since we don't have a separate 'manager' model.